File: src/main/java/ru/jpanda/jenkinsci/plugins/telegrambot/telegram/commands/HelpCommand.java
package ru.jpanda.jenkinsci.plugins.telegrambot.telegram.commands;

//import org.telegram.telegrambots.api.methods.send.SendMessage;
//import org.telegram.telegrambots.api.objects.Chat;
//import org.telegram.telegrambots.api.objects.User;
//import org.telegram.telegrambots.bots.AbsSender;
//import org.telegram.telegrambots.exceptions.TelegramApiException;
//import org.telegram.telegrambots.logging.BotLogger;

import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
import org.telegram.telegrambots.meta.api.objects.Chat;
import org.telegram.telegrambots.meta.api.objects.User;
import org.telegram.telegrambots.meta.bots.AbsSender;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
import org.telegram.telegrambots.meta.logging.BotLogger;

public class HelpCommand extends AbstractBotCommand {

    private static final String LOG_TAG = "/help";

    public HelpCommand() {
        super("help", "command.help");
    }

    @Override
    public void execute(AbsSender absSender, User user, Chat chat, String[] strings) {
        SendMessage answer = new SendMessage();
        answer.setChatId(chat.getId().toString());
        answer.setText(botStrings.get("message.help"));

        try {
            absSender.execute(answer);
        } catch (TelegramApiException e) {
            BotLogger.error(LOG_TAG, e);
        }
    }
}

File: src/main/kotlin/ru/jpanda/jenkinsci/plugins/telegrambot/telegram/commands/HelpCommand.kt
package ru.jpanda.jenkinsci.plugins.telegrambot.telegram.commands

import org.telegram.telegrambots.meta.api.methods.send.SendMessage
import org.telegram.telegrambots.meta.api.objects.Chat
import org.telegram.telegrambots.meta.api.objects.Message
import org.telegram.telegrambots.meta.api.objects.User
import org.telegram.telegrambots.meta.bots.AbsSender
import org.telegram.telegrambots.meta.exceptions.TelegramApiException
import org.telegram.telegrambots.meta.logging.BotLogger

class HelpCommand: AbstractBotCommand {
    private val LOG_TAG = "/help"

    constructor():super("help", "command.help")

    override fun execute(absSender: AbsSender, user: User, chat: Chat, strings: Array<String>) {
        val answer = SendMessage()
        answer.chatId = chat.id!!.toString()
        answer.text = botStrings["message.help"]

        try {
            absSender.execute<Message, SendMessage>(answer)
        } catch (e: TelegramApiException) {
            BotLogger.error(LOG_TAG, e)
        }

    }
}